feature-group-definitions/async-clipboard.yml

```diff
@@ -1,4 +1,5 @@
 name: Async clipboard
+description: The Clipboard API provides asynchronous read and write access to the content of the system clipboard, by requesting permission from the user.
```

Comment:
Riffing on @foolip here:

```suggestion
description: The `navigator.clipboard` API asynchronously reads and writes to the system clipboard.
```

Reply:
> I think we should avoid the spec name unless it is also the best name for the feature.

I agree that for this particular case, "Clipboard API" is probably not the best name for the feature. At least I've not really heard developers talk about it this way. But otherwise, I think it's pretty common for devs to refer to features by their spec names rather than by the name of the main interface/method/property of the feature. 
Guess it will have to be on a case by case basis.